sockets.h

```diff
@@ -13,19 +13,20 @@ void printSocketBuff() {
 
 bool readSocket(WiFiClient& client, uint8_t dataType) {
   
-  bool res = false;
-  memset(socketBuff, '\0', MAX_SOCKET_BUFF_SIZE);
-  char* tmp = nullptr;
+    bool res = false;
+    memset(socketBuff, '\0', MAX_SOCKET_BUFF_SIZE);
+    char* tmp = nullptr;
   
-  if(client && client.connected()) {
+    if (client.connected()) {
     
     Serial.println("Client Connected");
-    delay(250);
+    delay(2500);
```

Comment:
do we really need so much time to wait here

Reply:
I have 1 button
I connect, send and disconnect at once
this is why it is strange I'm waiting for 2.5 second here

Reply:
I have 3 btns for this stuff)
This is why there is 2.5s delay.
Is it correct to have one btn for this?
If yes - I`ll rewrite my qt app to do this stuff when 1 btn is clicked.

Reply:
ok
3 buttons is definetely too much for that :)
tcp dialog looks like following for us

server: ESP-32 module
client: Ionic or Qt (?) App

client: 
      //(shows loading box)
      const socketId = await TcpSockets.create();
      await TcpSockets.connect(
        socketId,
        '192.168.4.1'
      );

server:  
      client.connected() == true

client: 
      const res = await TcpSockets.send(
        socketId,
        ArrayConverter.str2ArrayBuffer(ssid, pwd)
      );

server:
     client.read(reinterpret_cast<uint8_t*>(tmp), msgLength);
     parse(...)
     client.write(WiFi.localIP().toString().c_str()); //(or error)
     client.stop();

client:
      //(handles result)
      TcpSockets.disconnect(socketId);
      TcpSockets.close(socketId);
      //(hides loading box)

this is normal user's experience

Reply:
Reduced delay to 0.75s.